### ISSUE TITLE
Learning Room?

### ISSUE THREAD

**Issue by CrypticNothings** [CrypticNothings/4Aurora#19]
Now I have a whole Learn how to Code page done but where can it be added in the website?

**Comment by CrypticNothings** [comment]
Also, have a quiz room that should be linked from the Learn to Code page. I'm thinking maybe have a learning/school page which has a new NavBar with subjects to choose from.

**Comment by CrypticNothings** [comment]
Ok, I think it should be the Arts & Crafts room. Redo the NavBar and put a Return to "Home" link on there to get out of that area on the website.

**Comment by zablon-oigo** [comment]
Hey, Please explain more, I might be able to tackle it

**Comment by CrypticNothings** [comment]
I don't want to seem rude or offend you but I'm not sure if I want help on this one. This is a bit personal to me and I feel it'd lose that special value if I outsource help on it. Does that sound dumb? Selfish? I appreciate you though. I feel you'll help me learn enough on the other one as is because it seems you have a completely different method you go about this.

**Comment by zablon-oigo** [comment]
Not at all! It's completely understandable. Some things are deeply personal, and maintaining that special value is important. I respect your feelings on this—it's not dumb or selfish. Feel free to explore and create at your own pace. I'm here whenever you need guidance or information.
